.gitattributes

```diff
@@ -1,2 +1,9 @@
 # Some storm-webapp logviewer tests require input files to have LF line endings due to byte counting.
-storm-webapp/src/test/resources/*.log.test text eol=lf
\ No newline at end of file
+storm-webapp/src/test/resources/*.log.test text eol=lf
+
+# There're reports of EOL conversion messing up PNG files, but that might have been a bug in git 2.10 only (see https://github.com/git/git/blob/master/Documentation/RelNotes/2.10.0.txt#L248 for details)
+*.png binary
```

Comment:
Could you try reversing the order of this and * text=auto? I'm pretty sure the latest command wins.

It looks like the failing tests are trying to load tarballs and zip files. Does adding *.tar.gz binary and *.zip binary fix it?

Reply:
also a .tgz file in there

Reply:
https://github.com/apache/storm/blob/master/storm-server/src/test/java/org/apache/storm/localizer/AsyncLocalizerTest.java#L493